privacyidea/config.py

```diff
@@ -95,6 +95,11 @@ class TestingConfig(Config):
                        "resolver": "resolverX"}]
 
 
+class FileAuditConfig(TestingConfig):
```

Comment:
We can avoid cluttering the config by using the [OverrideConfigTestCase](https://github.com/privacyidea/privacyidea/blob/a8dd0b5a093e38c530fbec66956729b000c38a47/tests/base.py#L200) class introduced by @fredreichbier